### ISSUE TITLE
bug(rule): unique-headings not ignored

### ISSUE THREAD

**Issue by Kristinita** [CSSLint/csslint#759]
### 1. Summary

I couldn't ignore [**`unique-headings` rule**](https://github.com/CSSLint/csslint/wiki/Headings-should-only-be-defined-once), use [**ignoring parts of CSS during linting**](https://github.com/CSSLint/csslint/wiki/Ignoring-parts-of-CSS-during-linting).

I don't reproduce this issue for another CSSLint rules.

### 2. Environment

+ Windows 10.0.18362 Enterprise 64-bit EN
+ Node.js 12.12.0
+ csslint 1.0.4

### 3. Data

+ `KiraIgnoreHeadings.css`:

```css
/* csslint ignore:start */

h1,
.SashaFirstClass {
    font-family: sans-serif
}

h1,
.SashaSecondClass {
    font-style: italic
}

/* csslint ignore:end */

```

### 4. Steps to reproduce

```text
csslint KiraIgnoreHeadings.css
```

I reproduce the problem for [**CSSLint online**](http://csslint.net/).

### 5. Expected behavior

No errors and/or warnings.

### 6. Actual behavior

```css
csslint: There is 1 problem in D:\SashaDebugging\KiraCSSLint\KiraIgnoreHeadings.css.

KiraIgnoreHeadings.css
1: warning
You have 2 h1s defined in this stylesheet.
```

### 7. Not helped

1. I can't use [**ignoring rules per line**](https://github.com/CSSLint/csslint/wiki/Ignoring-parts-of-CSS-during-linting#ignoring-rules-per-line), because I use [**Stylus**](http://stylus-lang.com/) → I can't preserve comment on specific line after Stylus to CSS compilation.
1. [**peterorum answer**](https://stackoverflow.com/a/35260823/5951529) doesn't work for me, see [**my comment**](https://stackoverflow.com/questions/29248792#comment103080092_35260823).

#### 8.1. “You don't need to ignore this option”

Please, read **#80**. I agree with users @trygve-lie and @Nateowami.

Thanks.
